### ISSUE TITLE
Is Mercator dead?

### ISSUE THREAD

**Issue by tobiasfeistmantl** [humanmade/Mercator#115]
Hi,

I'm wondering if Mercator is still in development? There are still many issues from last year open and plugins like WP Ultimo have them inside.

Thanks!

**Comment by tobiasfeistmantl** [comment]
That's true. Maybe they do. I was just wondering if this project is still in maintenance. The guys from humanmade probably use it for their client projects too. So for me the question is if the project just doesn't need any modifications or if they've dumped it or work with something else.

**Comment by jfeliweb** [comment]
I get where you are coming from... They will probably get back to it when they slow down on their [Altis](https://www.altis-dxp.com/) project. I guess they are using this in there since they offer a multisite feature.

**Comment by rmccue** [comment]
Mercator's still in use on some of our production sites, and we continue to maintain as needed for our purposes pursuant to our [support policy for open source](https://engineering.hmn.md/projects/support/).

> I guess they are using this in there since they offer a multisite feature.

We don't bundle Mercator with Altis, as it's not a requirement for using multisite; it's just a "domain aliasing" feature.
